### ISSUE TITLE
Particle System Plugin

### ISSUE THREAD

**Issue by yikuansun** [photopea/photopea#3532]
Hi,

I have made a plugin for particle simulation like those in cinematic VFX softwares. It can be customized to get different effects. For example, here are some different outputs I can get from playing around with the settings:
<img width="1126" alt="explosion" src="https://user-images.githubusercontent.com/38194363/125140144-dcb35080-e0df-11eb-84b3-3f3572466c50.PNG">
<img width="1062" alt="fire" src="https://user-images.githubusercontent.com/38194363/125140152-dfae4100-e0df-11eb-9115-13613dd3ba04.PNG">
<img width="1113" alt="spell" src="https://user-images.githubusercontent.com/38194363/125140156-e2109b00-e0df-11eb-95f9-9d65508acc7a.PNG">

I was wondering if you could add it to Photopea.

The JSON data is:
```
{
    "name"  : "Particle System",
    "url"   : "https://yikuansun.github.io/photopea-particlesystem",
    "width" : 400,
    "height": 500,
    "icon"  : "https://yikuansun.github.io/photopea-particlesystem/icon.svg"
}
```

**Comment by photopea** [comment]
Ok, I have added it.

Do you have any statistics on how many people are using your plugins?

BTW. the Width and the Height of a plugin are not needed now, as these windows are resizable in Photopea.

**Comment by yikuansun** [comment]
> Ok, I have added it.

Thanks!

> Do you have any statistics on how many people are using your plugins?

No, I don't have any analytics yet.  I could add analytics if they are important to you, and if I can find a free analytics service.
On 3D Elements, I see people are starting to submit models (and some spam).

> BTW. the Width and the Height of a plugin are not needed now, as these windows are resizable in Photopea.

Yeah, I realized that. I kept the attributes there for no particularly good reason, I guess I just didn't feel like taking them out. I only realized today that the API guide doesn't say width and height anymore. I'll leave them out next time :)
